keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -2424,7 +2551,7 @@ There will be pods which have a non-empty PodSpec.ResourceClaims field and Resou
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 For kube-controller-manager, metrics similar to the generic ephemeral volume
```

Comment:
Are there any metrics we can add to scheduler to better understand some internal steps?

Even just a simple counter for creating the new API objects (like PodScheduling) could be useful. Or a gauge for the number of pods that are waiting for updates in the objects before being scheduled.

Reply:
> Even just a simple counter for creating the new API objects (like PodScheduling) could be useful.

I agree, this seems useful. I'll add that here and will implement for 1.29.

> Or a gauge for the number of pods that are waiting for updates in the objects before being scheduled.

I think this is covered by ["unschedulable_pods"](https://github.com/kubernetes/kubernetes/blob/6f5fa2eb2f4dc731243b00f7e781e95589b5621f/pkg/scheduler/metrics/metrics.go#L200-L206). Let me describe that here.

Reply:
> I agree, this seems useful. I'll add that here and will implement for 1.29.

I just remembered why I didn't already add those: the same information about "PodSchedulingContexts created/updated" is already available through the apiserver metrics. IMHO there is no need to also record the same thing on the client side. The only difference would be that a "failure" count could also include requests that don't even reach the apiserver. Not sure whether that's worth it?

For now I'll add documentation about what exists already.